### PR TITLE
code-owners for App

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

updating codeowners for examples and source and tests and docs

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @tchaton @rohitgr7

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -41,6 +45,11 @@
 /src/pytorch_lightning/tuner                @SkafteNicki @borda @awaelchli
 /src/pytorch_lightning/utilities            @borda @tchaton @SeanNaren @carmocca
 
+/src/pytorch_app                           @tchaton @awaelchli @manskx @hhsecond
```
Comment (awaelchli) [inline .github/CODEOWNERS:48]: what about the rest of the codeowners file from lightning_app?

Comment (tchaton) [reply .github/CODEOWNERS:48]: Yes, can be more exhaustive there and take exactly the code owner file on app ?

Comment (Borda) [reply .github/CODEOWNERS:48]: @awaelchli good catch, thx
```suggestion
/src/lightning_app                           @tchaton @awaelchli @manskx @hhsecond
```

**Comment by manskx** [comment]
Can we add code-owrners for tests also?

**Comment by Borda** [comment]
> Can we add code-owrners for tests also?

Pls make a suggestion 🐰

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -41,6 +45,11 @@
 /src/pytorch_lightning/tuner                @SkafteNicki @borda @awaelchli
 /src/pytorch_lightning/utilities            @borda @tchaton @SeanNaren @carmocca
 
+/src/lightning_app                           @tchaton @awaelchli @manskx @hhsecond
+
+# Examples
+/examples/app_*                            @tchaton @awaelchli @manskx @hhsecond
+
```
Comment (manskx) [inline .github/CODEOWNERS:52]:
```suggestion

/tests/tests_app                          @tchaton @awaelchli @manskx @hhsecond
/tests/tests_app_examples                 @tchaton @awaelchli @manskx @hhsecond

```

**Review by manskx — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (awaelchli) [reply .github/CODEOWNERS]: @Borda can we please add the codeowners from lightning_app as I suggested?
https://github.com/Lightning-AI/lightning-app/blob/master/.github/CODEOWNERS
I don't want to be code owner for everything.

**Comment by awaelchli** [comment]
Please see my comment https://github.com/Lightning-AI/lightning/pull/13497#discussion_r912260395

File: .github/CODEOWNERS (hunk 1 above)
Comment (Borda) [reply .github/CODEOWNERS]: Sure, apology for fast merge... Pls send pr with correction ;)

**Pull request merged**